bigtable/quickstart/main.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START bigtable_quickstart]
+import argparse
+
+from google.cloud import bigtable
+
+
+def main(project_id, instance_id, table_id):
```

Comment:
Even though you don't use them, I think it'd be helpful to show default values so that people can see what a project ID, instance ID, table ID look like.

```
def main(project_id="your-project-id", instance_id="your_instance", table_id="your_table"):
```

Reply:
Done in 56393fe